drivers/SmartThings/jbl/src/jbl/device_manager.lua

```diff
@@ -0,0 +1,144 @@
+local log = require "log"
+local json = require "st.json"
+local fields = require "fields"
+
+local capabilities = require "st.capabilities"
+
+local device_manager = {}
+device_manager.__index = device_manager
+
+local function is_new_audioTrackData(device, audioTrackData)
+    local latestTrackData = device:get_latest_state("main", capabilities.audioTrackData.ID, capabilities.audioTrackData.audioTrackData.NAME, "")
```

Comment:
Since this is expected to be an object if populated, I think `{}` or `nil` is more appropriate for the default value